app/helpers/resource_fetch_helper.rb

```diff
@@ -103,6 +105,19 @@ def update_resource(fhir_resource)
     raise TIMEOUT_ERROR_MESSAGE
   end
 
+  def create_resource(fhir_resource)
+    response = client.create(fhir_resource)
```

Comment:
should we check if fhir_resource is nil?

Reply:
Updated